cypress/support/script-tag/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+
+  <meta name="description" content="Open Health Imaging Foundation DICOM Viewer" />
+  <meta name="viewport"
+    content="width=device-width,initial-scale=1.0,minimum-scale=1,maximum-scale=1,user-scalable=no" />
+  <meta name="theme-color" content="#000000" />
+  <meta http-equiv="cleartype" content="on" />
+  <meta name="MobileOptimized" content="320" />
+  <meta name="HandheldFriendly" content="True" />
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
```

Comment:
I think we removed the dependency on this, but we should check.

Reply:
We removed this dependency in `react-viewerbase` so far. The `Viewers` still uses some bootstrap classes and when I remove the import link, I notice some changes in the layout but no breaks. Should I remove then?

Reply:
Ah shoot. It would be nice if we can resolve this in a separate PR. I would like our bootstrap dependency to be dead.